MDE-EdFiClientSDK/EdFi/generate-sdk.ps1

```diff
@@ -22,9 +22,8 @@ $params = @(
 )
 
 & $javaExecutable $params -i "$metaDataUrl/data/v3/descriptors/swagger.json"                                               --api-package Apis.Descriptors                                                  --model-package Models.Descriptors
-& $javaExecutable $params -i "$metaDataUrl/identity/v2/swagger.json"                                                       --api-package Apis.Identity                                                     --model-package Models.Identity
```

Comment:
Would it be better to comment this out instead of removing, i.e. 

\# Removed according to MOAI-800
\# & $java.......